src/components/VHeader/README.md

```diff
@@ -1,11 +1,17 @@
 #VHeader
 
-The sticky header at the top of the page. On the homepage and the content pages, it shows the
-Openverse logo and page links ??and Search bar??, and on the Search page it additionally has
-a Content Switcher, "burger menu" for page links, and a filters button.
-On mobile, the header only shows the logo and close button when the overlay (pages, content
-switcher, or filters) is open.
+The sticky header at the top of the page. On the homepage and the content pages, it shows one version, and on the Search page it additionally has a Content Switcher, collapsed menu for page links, and a filters button. On mobile, the header only shows the logo and close button when the overlay/flyout menu (containing pages, the content switcher, and filters) is open.
 
 ## Components
+
 ### VLogoLoader
+
+- TBD
+
 ### VFilterButton
+
+This button toggles the filter drawer on desktop and opens the mobile menu set to the filters pane on mobile.
+
+- When the header is scrolled, remove the button's border
+- When there's applied filters, replace the filter icon with the # of active filters
+- On scroll, hide the word label (the word 'Filters') to save space
```

Comment:
The word is hidden on scroll for me, and showed again on top, but I agree is a bit odd to have only the number, maybe it is worth leaving the icon also?

Reply:
I agree this could be a concern, it's only informed by limited horizontal space to fit everything into a single header on mobile. The search bar will be in the middle.

The word 'Filters' is _only_ removed after the user scrolls, so they'll at least see it initially if they're paying attention. Adjusting things so the icon fits in mobile after scrolling could be a really good compromise, but I'll leave figuring this out to @panchovm

Reply:
Agree there is a risk in removing words and relying solely on a numeric symbol, but for this release, I think it's a good solution for the navigation on mobile. Yet, I would follow the behavioral statistic to assess its success after a few months.

Reply:
Another thing to keep in mind about the mobile version is that there will also be the WordPress header hamburger menu present on the page at the same time. I actually already made the error of confusing the two 😭

Reply:
Yes 💔 . We are aware of the double header navigation risk and, hopefully, we can work on a new WordPress header this year. Although, the scroll just hides the WP header whereas Openverse header remains fixed.